seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/JsonDeserializationSchema.java

```diff
@@ -132,11 +133,11 @@ private SeaTunnelRow convertJsonNode(JsonNode jsonNode) {
         }
         try {
             return (SeaTunnelRow) runtimeConverter.convert(jsonNode);
-        } catch (Throwable t) {
+        } catch (RuntimeException e) {
```

Comment:
I prefer change `Throwable` to `Exception` of all catch condition. Change to `RuntimeException` would lose some exception like `NullPointerException`.

Reply:
I prefer to `RuntimeException`. Because there are no checked exceptions here, and `RuntimeException` can cover all conditions.

Reply:
make sense to me.